.github/workflows/gradle-build-pr.yml

```diff
@@ -26,155 +26,72 @@ jobs:
     steps:
       - name: "Checkout code"
         uses: actions/checkout@v2
-      - name: "Install Java"
-        uses: actions/setup-java@v1
+      - name: "Run Connected test via Gradle"
+        uses: broadinstitute/datarepo-actions@0.1.0
         with:
-          java-version: 8
-      - name: "Install Gradle"
-        uses: broadinstitute/gradle-command-action@v1
-      - name: "Install Google Cloud SDK"
-        uses: GoogleCloudPlatform/github-actions/setup-gcloud@0.1.1
+          actions_subcommand: 'gradleinttest'
+          pgport: ${{ job.services.postgres.ports[5432] }}
+          test_to_run: 'testConnected'
+          role_id: ${{ secrets.ROLE_ID }}
+          secret_id: ${{ secrets.SECRET_ID }}
+  deploy_test_integration:
+    timeout-minutes: 180
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    if: "!contains( github.event.pull_request.labels.*.name, 'skip-ci')"
+    runs-on: ${{ matrix.os }}
+    ## skips if pr label is 'skip-ci'
+    # run a local Postgres container in Docker for the basic check tests
+    services:
+      postgres:
+        image: postgres:9.6
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v2
+      - name: "Whitelist Runner IP"
+        uses: broadinstitute/datarepo-actions@0.1.0
```

Comment:
We repeat this version string a lot of places. Is there a way we can define it once and then reuse that throughout this file?

Reply:
version string ??? you mean the tag 0.1.0?